build_tools/ci/cpu_comparison/run_test.py

```diff
@@ -449,33 +403,22 @@ def run_all(argv):
        ./matmul_template. For example you might want to create ./conv_template
     """
 
-    assert_num_args(argv)
-
-    output_dir = os.path.realpath(argv[0])
-    os.makedirs(output_dir, exist_ok=True)
-    validate_directory(output_dir)
-
-    iree_install_dir = os.path.realpath(argv[1])
-    validate_directory(iree_install_dir)
+    if output_dir.exists():
+        shutil.rmtree(output_dir)
```

Comment:
This is dangerous! What if the user has something in output_dir that they don't want removed, for example maybe they accidentally provide the path to their home directory for output_dir.

Reply:
is it more or less dangerous than the issue you had a little while ago where tests were passing even though nothing was compiling? i'm doing this by hand right now (ie in an outer script). anyway i can take it out and leave it to the user to make the decision.

Reply:
Maybe add a script flag which by default is "do not remove"

Reply:
too many script flags turns into "programming by script flags". i'll just leave it out.

Reply:
Ok. Yeah I'm not sure what a good solution is, sorry. Maybe walk the dir and remove xclbins, that would avoid the issue we had before when compilation was failing (although is seems arbitrary and complex).